doc/releases/migration-guide-3.7.rst

```diff
@@ -381,6 +381,23 @@ Display
             compatible = "sitronix,st7735r";
             reg = <0>;
             spi-max-frequency = <32000000>;
+
+Crypto
+======
+
+* The CSS driver has been deprecated on NXP lpc55s36
+
+Ethernet
+========
+
+* Deprecated NXP's eth_mcux and introduced nxp_enet
+
+USB
+===
+
+* Introduction of a USB device controller driver for NXP SoC's (udc_mcux)
```

Comment:
> Put these into already existing sections rather than adding you owns

Existing sections? I don't see one for each of these. But perhaps I am not looking at the file correctly.